src/dotnet/APIView/APIViewWeb/Client/src/pages/index.ts

```diff
@@ -48,6 +49,9 @@ $(() => {
       uri = uri + '&type=' + encodeURIComponent(`${$(this).val()}`);
     });
 
+    if (notApprovedForFirstRelease.prop("checked"))
+      uri = uri + '&notApprovedForFirstRelease=true';
```

Comment:
In general I don't recommend single line if statements as it makes it too easy for bugs to creep in.

```suggestion
    if (notApprovedForFirstRelease.prop("checked")) {
      uri = uri + '&notApprovedForFirstRelease=true';
    }
```